compiler/optimizer/ValuePropagationCommon.cpp

```diff
@@ -1159,6 +1158,7 @@ void OMR::ValuePropagation::transformArrayCopyCall(TR::Node *node)
         srcOffHigh >= 0 && dstOffHigh >= 0 && copyLenHigh >= 0)
       {
       transformTheCall = true;
```

Comment:
We might not have any type information about either array. Do you still want this default in that case? Maybe it should default to something like `primitiveTransform && referenceTransform`.